### DISCUSSION TITLE
Documentation TimeSeriesForecaster and TimeSeriesClassifier

### DISCUSSION THREAD

**Discussion by hjannikh** [keras-team/autokeras#1751] [Q&A] [answered]
Hello,
Is there a documentation for the TimeSeriesForecaster and TimeSeriesClassifier? Unfortunately, I cannot find this on the AutoKeras website.

**Comment by haifeng-jin** [comment, marked as answer]
The `TimeSeries` related tasks are still in an experimental phaze. We do not have a tutorial for the classifier yet.
